operators/distiller-state-client/Containerfile

```diff
@@ -1,11 +1,11 @@
 FROM ghcr.io/nersc/interactem/operator
 
 WORKDIR /app
-COPY ./distiller-pipeline/pyproject.toml ./distiller-pipeline/poetry.lock ./distiller-pipeline/README.md /app/
+COPY ./distiller-streaming/pyproject.toml ./distiller-streaming/poetry.lock ./distiller-streaming/README.md /app/
 
 RUN poetry install --no-root
 
-COPY ./distiller-pipeline/distiller_pipeline/ /app/distiller_pipeline/
+COPY ./distiller-streaming/distiller_streaming/ /app/distiller_streaming/
```

Comment:
Does it make sense to have a `ghcr.io/nersc/interactem/distiller-streaming` base container to build all the distiller streaming operators from, rather than doing the copy in each of the container files?

Reply:
yes, done